code/src/css/todoItem.css

```diff
@@ -0,0 +1,62 @@
+.todo-item {
+  padding-left: 15px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #eee;
+  color: #063053;
+  margin-bottom: 15px;
+  border: 1px solid #bebebe;
+  box-shadow: 3px 5px 8px 1px rgba(0, 0, 0, 0.2);
+  transition: box-shadow 500ms ease;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+}
+
+.todo-item:hover {
+  box-shadow: 3px 8px 16px 1px rgba(0, 0, 0, 0.3);
+}
+
+.complete-checkbox {
+  width: 25px;
+  height: 25px;
+  cursor: pointer;
+  margin-right: 15px;
+}
+
+.complete-checkbox input {
+  display: none;
+}
+
+.checkmark {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #063053;
+  border-radius: 3px;
+  display: inline-block;
+  background: #5c7c99 url("../img/checked.png") center/0 no-repeat;
+  transition: background-size 300ms ease-in;
+}
```

Comment:
If you write a long todo, the checkbox shrinks👇
![Skärmavbild 2020-12-21 kl  13 45 29](https://user-images.githubusercontent.com/51529201/102778403-cbe4fc80-4392-11eb-8eac-77bb928c566b.png)